tools/sifgen/common/types/genesis.go

```diff
@@ -175,6 +175,20 @@ type StakingParams struct {
 	BondDenom         string `json:"bond_denom"`
 }
 
+type Mint struct {
+	Minter interface{} `json:"minter" yaml:"minter"`
```

Comment:
Same for the minter ,in case the default value 
sdk.NewDecWithPrec(13, 2) is not okay we need to add it to sifgen tool

Reply:
Yup, all default for now